training/nvidia-bootc/Containerfile

```diff
@@ -178,7 +178,7 @@ RUN for i in /usr/local/bin/ilab*; do \
 # Added for running as an OCI Container to prevent Overlay on Overlay issues.
 VOLUME /var/lib/containers
 
-RUN if [ -f "/run/.input/ilab" ]; then \
```

Comment:
Should check
 /run/.input/instructlab-nvidia

Reply:
Could you do the same with intel-bootc and amd-bootc?

Reply:
Sure. Thank you Dan.